src/fiobank/fiobank.py

```diff
@@ -47,8 +48,10 @@ def __init__(self, token: str, *, decimal: bool = False):
             )
             self.float_type = float
 
-        # http://www.fio.cz/xsd/IBSchema.xsd
-        self.transaction_schema = {
+    @property
+    def transaction_schema(self) -> dict:
```

Comment:
I know this doesn't start with underscore, but I don't consider this to be a part of the public API. I think we should remove it, and `self.info_schema` as well. If you think people could use it, we can have it there as a property which raises NotImplementedError and warns the user they're using undocumented and non-public interface, which has been removed.